package-lock.json

```diff
@@ -178,7 +178,7 @@
         "tap": "^16.0.1"
       },
       "engines": {
-        "node": "^12.13.0 || ^14.15.0 || >=16"
+        "node": "^14.17.0 || ^16.0.0 || >=18.0.0"
```

Comment:
why 14.17 and not 14.15, since 14.15 is when it entered LTS?

also why 16.0.0 and not v14.13.0 when it entered LTS?

Reply:
Mostly as an attempt to align with other packages we consume in the ecosystem. I wanted to set our engines as low as we could, while still allowing for packages like `readable-stream@4` to be installed (which has its engines field set to [`^12.22.0 || ^14.17.0 || >=16.0.0`](https://github.com/nodejs/readable-stream/blob/77e0378b1a8a46c42fc3fce43969b83abf69ca36/package.json)).

Same logic applies to 16. I didn't see any reason to raise this floor since we wont be able to consume any packages that changed to `> 16.0.0` until `npm10` when this will all get revisited.

Reply:
This is a summary of https://github.com/npm/statusboard/issues/519, which goes through the other engines values we discussed.

Reply:
After typing this out I now realize that a dependency could ship with `^14.17.0 || ^16.13.0 || >=18` and we couldn't be able to consume it so maybe there is a case for raising the floor for 16.

Reply:
The engines is now `^14.17.0 || ^16.13.0 || >=18`